src/lib/logger.ts

```diff
@@ -106,6 +113,39 @@ export class Logger {
     Logger.log(event, parameters);
   }
 
+  public static logToolChange(change: ToolChangeEventType, toolId: string) {
```

Comment:
Seems like we should probably add a `toolType` argument here to avoid having to figure it out ourselves.

Reply:
I agree. Added.